modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManagerView.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <table class="apos-table">
```

Comment:
This template was copied almost without changes from the pieces manager component.